src/nunavut/lang/c/templates/Header.j2

```diff
@@ -1,33 +1,35 @@
-/*
- *
- * UAVCAN data structure definition.                                                                          +-+ +-+
- *                                                                                                            | | | |
- * AUTOGENERATED, DO NOT EDIT.                                                                                \  -  /
- *                                                                                                              ---
- *                                                                                                               o
- * +------------------------------------------------------------------------------------------------------------------+
- *
- * Generator:
- * nunavut-{{ nunavut.version }} (serialization was {% if nunavut.support.omit %}not {% endif %}enabled)
- *
- *
- * Source File:
- * {{ T.source_file_path }}
- *
- * Template:
- * {{ self._TemplateReference__context.name }}
- *
- * Generated at:  {{ now_utc }} UTC
- * Is deprecated: {{ T.deprecated and 'yes' or 'no' }}
- * Fixed port ID: {{ T.fixed_port_id }}
- * Full name:     {{ T.full_name }}
- * Version:       {{ T.version.major }}.{{ T.version.minor }}
- *
- */
+// UAVCAN data structure definition.                                                                          +-+ +-+
```

Comment:
why did we switch the comment style?

Reply:
https://kb.zubax.com/pages/viewpage.action?pageId=2195699#ZubaxC++codingconventions-Comments

Reply:
💩